### PR TITLE
Grid tests

### PR DESCRIPTION
**Pull request by joakim-hove**
This should fix one test. One test is still failing.

I don't really understand this; but it seems: https://github.com/Ensembles/ert/pull/1404 was merged prematurely. When this is merged we should have exactly one failing test - related to this,

### CODE REVIEW

File: libecl/tests/statoil_tests.cmake [hunk 1]
```diff
@@ -62,7 +62,7 @@ add_test( ecl_grid_cell_contains4 ${EXECUTABLE_OUTPUT_PATH}/ecl_grid_cell_contai
 
 add_executable( ecl_grid_cell_contains_wellpath ecl_grid_cell_contains_wellpath.c )
 target_link_libraries( ecl_grid_cell_contains_wellpath ecl test_util )
-add_test( ecl_grid_cell_contains_wellpath1 ${EXECUTABLE_OUTPUT_PATH}/ecl_grid_cell_contains_wellpath ${PROJECT_SOURCE_DIR}/test-data/Statoil/ECLIPSE/CellContains/model/SMS-0.EGRID ${PROJECT_SOURCE_DIR}/test-data/Statoil/ECLIPSE/CellContains/R_PB-4H.txt )
+add_test( ecl_grid_cell_contains_wellpath1 ${EXECUTABLE_OUTPUT_PATH}/ecl_grid_cell_contains_wellpath ${PROJECT_SOURCE_DIR}/test-data/Statoil/ECLIPSE/CellContains/model/SMS-0.EGRID ${PROJECT_SOURCE_DIR}/test-data/Statoil/ECLIPSE/CellContains/R_PB-4H.jira )
```
Comment (pgdr) [inline libecl/tests/statoil_tests.cmake]: why `.jira`?

Comment (joakim-hove) [inline libecl/tests/statoil_tests.cmake]: Just becase the data was Copy & Pasted from the Jira task.

File: libecl/tests/ecl_grid_cell_contains_wellpath.c [hunk 2]
```diff
@@ -44,14 +44,13 @@ vector_type * load_expected( const char * filename ) {
   vector_type * expected = vector_alloc_new();
 
   while (true) {
-    double x,y,z,md;
+    double x,y,z;
     int i,j,k;
 
-    if (fscanf( stream , "%lg %lg %lg %lg %d %d %d" , &x,&y,&md,&z,&i,&j,&k) == 7) {
+    if (fscanf( stream , "%lg %lg %lg %d %d %d" , &x,&y,&z,&i,&j,&k) == 6) {
```
Comment (pgdr) [inline libecl/tests/ecl_grid_cell_contains_wellpath.c]: has `md` been removed from the file or was it never there?  Isn't it a crucial part of a wellpath?

**Comment by joakim-hove** [comment]
OK - master is broken at the moment; with this PR it is less broken. I'll merge. Fixing the real problem might be on-trivial - dang;

**Pull request merged**